pyproject.toml

```diff
@@ -88,6 +88,8 @@ sphinxcontrib-mermaid = "==0.7.1"
 furo = "==2022.6.21"
 sphinx-reredirects = "==0.1.1"
 nbsphinx = "==0.8.9"
+xgboost = "==1.6.0"
+matplotlib = "==3.5.2"
```

Comment:
Those dependencies should only be added to the `quickstart_xgboost_horizontal` folder.